packages/camera/src/setUpTests.js

```diff
@@ -0,0 +1,5 @@
+import { jest } from '@jest/globals';
+import '@testing-library/jest-dom';
+
+// import '@testing-library/jest-native/extend-expect';
```

Comment:
remove this comment. Should you not import `import '@testing-library/jest-native';` here?

Reply:
its already imported in jest.config.js on line no.22